app/controllers/accredited-bodies.js

```diff
@@ -126,13 +126,13 @@ exports.edit_accredited_body_permissions_post = (req, res) => {
       accreditedBody
     })
 
-    req.flash('success', 'Accredited body permissions updated')
+    req.flash('success', 'Accredited provider permissions updated')
     res.redirect(`/organisations/${req.params.organisationId}/cycles/${req.params.cycleId}/accredited-bodies`)
   }
 }
 
 /// ------------------------------------------------------------------------ ///
-/// NEW ACCREDITED BODY
+/// NEW ACCREDITED PROVIDE
```

Comment:
There's a typo here. Says 'Provide' not 'Provider'

Reply:
Eagle eyes. In this instance, it's just a code comment. But I'll update.